modules/odc-creating-a-visual-connection-between-components.adoc

```diff
@@ -0,0 +1,26 @@
+:_content-type: PROCEDURE
+[id="odc-creating-a-visual-connection-between-components_{context}"]
+== Creating a visual connection between components
+
+You can depict an intent to connect application components using the visual connector.
```

Comment:
Please review the 21 instances of "using" in this PR and update them in accordance with the following IBM Style Guide guideline about "using":

To avoid ambiguity, replace this gerund with either "by using" or "that uses".
To refer to something "using" one thing or another is often ambiguous. For example, the sentence "Back up the server using
ABC" might mean "Back up the server by using ABC" or "Back up the server that uses ABC". This advice also applies to gerunds
other than "using". For more information, see the style and grammar tips in the Global audiences topic in IBM Style and the
section about "Clarity" in "Developing Quality Technical Information".

Reply:
I have replaced "using" gerund in this PR  with either "by using" or "that uses" where it is feasible. Thank you!